fivetran/rds-writeonly-role.sql

```diff
@@ -0,0 +1,27 @@
+-- creates a write-only role for fivetran to connect to RDS for specific tables
+DO
+$$
+  DECLARE
+    table_name     varchar;
+    project        varchar   := '';
+    environment    varchar   := '';
+    allowed_tables varchar[] := array [
+      'table1',
+      'table2'
+      ];
+    password       varchar   := 'my-super-secret-password';
+  BEGIN
+    EXECUTE format('CREATE ROLE %s_%s_writeonly_fivetran;', project, environment);
+    EXECUTE format('ALTER ROLE %s_%s_writeonly_fivetran WITH LOGIN ENCRYPTED PASSWORD ''%s'';', project, environment,
+                   password);
+    EXECUTE format('GRANT USAGE ON SCHEMA public TO %s_%s_writeonly_fivetran;', project, environment);
+    FOREACH table_name IN ARRAY allowed_tables
+      LOOP
+        EXECUTE format('GRANT ALTER ON TABLE %s TO %s_%s_writeonly_fivetran', table_name, project, environment);
+        EXECUTE format('GRANT CREATE, UPDATE ON TABLE %s TO %s_%s_writeonly_fivetran', table_name, project,
+                       environment);
```

Comment:
```suggestion
        EXECUTE format('GRANT INSERT, SELECT, UPDATE, DELETE, ALTER ON TABLE %s TO %s_%s_writeonly_fivetran', table_name, project, environment);
```

we can summarize these. We'll also need delete & select so the sync can correctly determine which records have been inserted and delete records that need to be removed.